src/back-end/test/create.test.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Tests export user creation functions
+ */
+
+/* Mock envvars */
+process.env.HASHKEY = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+
+/* Imports */
+const create = require("../createFiles/createUser");
+const schema = require("../schema.js");
+const mongoose = require("mongoose");
+
+describe("makeid() Test", () => {
+    test("Function produces unique IDs", () => {
+        const id1 = [create.makeid];
+        const id2 = [create.makeid];
+        const id3 = [create.makeid];
+        expect(id1).not.toBe(id2);
+        expect(id1).not.toBe(id3);
+        expect(id2).not.toBe(id3);
+    });
+});
+
+describe("createUser() Tests", () => {
+
+    /**
+     * Checks for an existing user
+     *
+     * @param {Object} userData Object containing email and password
+     * @callback (response) Sends either true or false based on whether the user exists
+     */
+    const checkUser = async (userData, callback) => {
```

Comment:
Rather than using a callback use a promise. There is no reason to have an async function with a callback.